packages/devtools/lib/src/timeline/frames_bar_chart.dart

```diff
@@ -23,6 +31,14 @@ class FramesBarChart extends CoreElement {
       ..paddingTop = '${topPadding}px';
 
     timelineController.onFrameAdded.listen((TimelineFrame frame) {
+      if (element.children.isEmpty) {
+        frameUIgraph = PlotlyDivGraph(this, frame, false); // Process chunks
```

Comment:
wouldn't it be more direct to check whether frameUiGraph != null
instead of 
element.children.isEmpty
?
(assuming the two are equivalent)

Reply:
I was simply copying the exact code that was checking for children.isEmpty I'll ask Kenzie if this is really necessary? @kenzieschmoll

Reply:
This is also code I left there from the original implementation. In the current version of the code, this is just to determine whether we should add the frame via `add(frameUI)` or `element.children.insert(0, frameUI.element)`. Which I don't think should matter if element.children is just empty and not null when it has zero children. I think checking for `frameUIGraph != null` makes more sense here.

Reply:
Great I'll make that change. Thx.